staging/src/k8s.io/mount-utils/mount_linux.go

```diff
@@ -409,6 +421,50 @@ func (mounter *Mounter) IsLikelyNotMountPoint(file string) (bool, error) {
 	return true, nil
 }
 
+// IsLikelyNotMountPoint determines if a directory is not a mountpoint.
+// It is fast but not necessarily ALWAYS correct. If the path is in fact
+// a bind mount from one part of a mount to another it will not be detected.
+// It also can not distinguish between mountpoints and symbolic links.
+// mkdir /tmp/a /tmp/b; mount --bind /tmp/a /tmp/b; IsLikelyNotMountPoint("/tmp/b")
+// will return true. When in fact /tmp/b is a mount point. If this situation
+// is of interest to you, don't use this function...
+func (mounter *Mounter) IsLikelyNotMountPoint(file string) (bool, error) {
```

Comment:
Please create (rename) two functions:
* `isLikelyNotMountPointStat`
* `isLikelyNotMountPointStatx`

And `IsLikelyNotMountPoint` then calls `isLikelyNotMountPointStatx` and falls back to `isLikelyNotMountPointStat` on `errStatxNotSupport`. Now IsLikelyNotMountPoint looks quite weird, with statx evaluation.

Reply:
fixed, thank you for you comment.